src/main/java/capstone/mfslbackend/controller/GameController.java

```diff
@@ -0,0 +1,40 @@
+package capstone.mfslbackend.controller;
+
+import capstone.mfslbackend.model.Game;
+import capstone.mfslbackend.response.dto.GamesResponse;
+import org.springframework.util.CollectionUtils;
+import capstone.mfslbackend.service.GameService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController()
+@RequestMapping("/player")
+public class GameController {
+    private final GameService gameService;
+    public GameController(GameService gameService) {
+        this.gameService = gameService;
+    }
+
+    @GetMapping("get-game")
```

Comment:
with the @ RequestMapping, everything will have "/game" added automatically, so for a simple get request, I don't think you need to add anything to the path